### ISSUE TITLE
The Android version does not support UTF-8

### ISSUE THREAD

**Issue by programandala-net** [smallbasic/SmallBASIC#82]
I've been trying the latest SmallBASIC (0.12.16) on Debian (9.11) and Android (5.0.1). It is exactly what I needed for a project (a text console program): A modern BASIC easy to install on Android, and which can interpret the same source developed and tried on Debian.

But I've realized that the Android version doesn't support UTF-8 sources (multi-byte characters are displayed apart as ISO 8859-1 characters). Encoding the source as ISO 8859-1 seems to work fine, but it's not enough for this project, which is multilingual.

Is this a bug, an oversight or by design?

**Comment by programandala-net** [comment]
It's not the Android version. I have compiled the SDL version on Debian, and the same problem happens.

**Comment by chrisws** [comment]
Yes, unfortunately it's a bug/oversight etc. It would probably be a lot of work to implement this so I'm not sure when I'd get time. Interestingly some people have made font type programs that draw characters using graphics primitives. It might be possible to use something like this as a workaround.

**Comment by ercanersoy** [comment]
I have tried using SmallBASIC on Microsoft Windows. SmallBASIC is very good project.

I have realized SmallBASIC not support UTF-8. I would like support UTF-8.
